kvbc/include/categorization/cmf/updates.cmf

```diff
@@ -0,0 +1,27 @@
+Msg MerkleUpdatesData 1 {
+    map string string kv
+    list string deletes
+}
```

Comment:
@dartdart26 @andrewjstone  Currently I added the CMF and the generated code files to a dedicated folder.
Personally, I'd rather generate the code and use it explicitly than generating it at build time.
What are your preferences for the generation and location of the files?

Reply:
Personally, I prefer that we generate them at build time to avoid:

- maintaining two dependent files (prone to error)
- changes to the compiler itself will not be reflected (i.e. a bug fix), unless manually rebuilt